tiledb/sm/encryption/encryption.cc

```diff
@@ -106,5 +106,41 @@ Status Encryption::decrypt_aes256gcm(
   STATS_FUNC_OUT(encryption_decrypt_aes256gcm);
 }
 
+Status Encryption::md5(ConstBuffer* input, Buffer* output) {
+  return md5(input, input->size(), output);
+}
+
+Status Encryption::md5(
+    ConstBuffer* input, uint64_t input_read_size, Buffer* output) {
+  return md5(input->data(), input_read_size, output);
+}
+
+Status Encryption::md5(
+    const void* input, uint64_t input_read_size, Buffer* output) {
+#ifdef _WIN32
+  return Win32CNG::md5(input, input_read_size, output);
+#else
+  return OpenSSL::md5(input, input_read_size, output);
+#endif
+}
+
+Status Encryption::sha256(ConstBuffer* input, Buffer* output) {
+  return sha256(input, input->size(), output);
+}
+
+Status Encryption::sha256(
+    ConstBuffer* input, uint64_t input_read_size, Buffer* output) {
+  return sha256(input->data(), input_read_size, output);
+}
+
+Status Encryption::sha256(
```

Comment:
Perhaps consider renaming `Encryption` to `Crypto`, since md5 and sha256 have to do with integrity, not encryption per se. Just a thought.

Reply:
If we do so, we will need to change `tiledb/sm/encryption/encryption.*` to `tiledb/sm/crypto/crypto.*`

Reply:
How about using 'Hash' or 'Digest' instead?

Reply:
That class implements encryption as well. So we either split into two classes (like `Encryption` and `Hash/Digest`), or we use one that describes both (like `Crypto`).

Reply:
Oh, I missed that. Disregard my comment in that case.

Reply:
I've renamed the package/class to crypto